docs/items/globallootmodifiers.md

```diff
@@ -0,0 +1,129 @@
+Global Loot Modifiers
+===========
+
+Global Loot Modifiers are a data-driven method of handling modification of harvested drops without the need to overwrite dozens to hundreds of vanilla loot tables or to handle effects that would require interactions with another mod's loot tables without knowing what mods may be loaded. Global Loot Modifiers are also stacking, rather than last-load-wins as modifications to loot tables would be.
+
+Registering a Global Loot Modifier
+-------------------------------
+
+You will need 3 things:
+1. Create a global_loot_modifiers.json file at /data/forge/loot_modifiers/
+	This will tell Forge about your modifiers and works similar to [tags].
+2. A serialized json representing your modifier
+    This will contain all of the data about your modification and allows data packs to tweak your effect.
+3. A class that extends `LootModifier`
+    The operational code that makes your modifier work and associated serializer.
+
+Finally, the serializer for your operational class is [registered] as any other `ForgeRegistryEntry`.
+
+[tags]: ../utilities/tags/tags.md
+[registered]: ../concepts/registries.md#registering-things
+
+The global_loot_modifiers.json
+-------------------------------
+
+All you need to add here are the registry names of your loot modifiers.
+```
+{
+  "replace": false,
+  "entries": [
+    "global_loot_test:silk_touch_bamboo",
+    "global_loot_test:smelting",
+    "global_loot_test:wheat_harvest"
+  ]
+}
+```
+
+`replace` causes the cache of modifiers to be cleared fully when this asset loads (mods are loaded in an order that may be specified by a data pack). For modders you will want to use `false` while data pack makers may want to specify their overrides with `true`.
+
+`entries` is an *ordered list* of the modifiers that will be loaded. Any modifier that is not listed will not be loaded and the ones listed are called in the order listed. This is primarily relevant to data pack makers for resolving conflicts between modifiers from separate mods.
+
+The serialized json
+-------------------------------
+
+This file contains all of the potential variables related to your modifier, including the conditions that must be met prior to modifying any loot as well as any other parameters your modifier might have. Avoid hard-coded values where ever possible so that data pack makers can adjust balance if they wish to.
+```
+{
```

Comment:
It would be better to also have the `type` field in the example file, since grabbing the type from the file name has been deprecated in 1.15 ([Source](https://github.com/MinecraftForge/MinecraftForge/blob/aa5b9d12ea35f6020d3f1d8117b6b4684740fc59/src/main/java/net/minecraftforge/common/loot/LootModifierManager.java#L131)) and plainly doesn't work in 1.14 ([Source](https://github.com/MinecraftForge/MinecraftForge/blob/90403d18f98db35d5f3980e29bb3bb8ab4965167/src/main/java/net/minecraftforge/common/loot/LootModifierManager.java#L131))

Reply:
👍 I wrote this stuff up before that change was made and forgot about it.

Reply:
Deleted that file and added the new page to navigation.